pkg/controllers/collaset/synccontrol/update.go

```diff
@@ -112,6 +124,25 @@ func attachPodUpdateInfo(ctx context.Context, pods []*collasetutils.PodWrapper,
 		podUpdateInfoList[i] = updateInfo
 	}
 
+	// attach replace info
+	var podUpdateInfoMap = make(map[string]*PodUpdateInfo)
+	for _, podUpdateInfo := range podUpdateInfoList {
+		podUpdateInfoMap[podUpdateInfo.Name] = podUpdateInfo
+	}
+	replacePodMap := classifyPodReplacingMapping(pods)
+	for originPodName, replacePairNewPod := range replacePodMap {
+		originPodInfo := podUpdateInfoMap[originPodName]
+		if replacePairNewPod != nil {
+			originPodInfo.isInReplacing = true
```

Comment:
I think `originPodInfo.isInReplacing` also should be true if `originPodInfo` has label `PodReplaceByReplaceUpdateLabelKey`.